scalableminds/webknossos
# Add gamma correction setting

See https://www.dfstudios.co.uk/articles/programming/image-programming-algorithms/image-processing-algorithms-part-6-gamma-correction/ for reference.

@normanrz maybe you have a dataset for which lends itself to test this feature?

I also tweaked the styling of the settings UI a bit. The most notable change is that the number inputs don't have borders, anymore. This makes the UI more lightweight and frees up some space.

### URL of deployed dev instance (used for testing):
- https://gammacorrection.webknossos.xyz/datasets/sample_organization/l4_sample/view#3584,3584,1024,0,1.3

### Steps to test:
- open a dataset (and an annotation)
- check that the gamma value defaults to 1 for all layers (segmentation layers shouldn't have that setting)
- play around with the gamma value

### Issues:
- fixes #6348 

------
(Please delete unneeded items, merge only when none are left open)
